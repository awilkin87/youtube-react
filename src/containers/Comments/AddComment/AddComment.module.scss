@use '../../../styles/shared.module';

.container {
  display: flex;
  align-items:flex-start;
  margin: 16px 0;

  .userAvatar {
    @include shared.left-avatar;
  }

  form {
    flex: 1;
  }
}
