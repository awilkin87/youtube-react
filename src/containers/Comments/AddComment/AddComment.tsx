import { Form, Image, TextArea } from 'semantic-ui-react';
import styles from './AddComment.module.scss';

export default function AddComment() {
  return (
    <div className={styles.container}>
      <Image className={styles.userAvatar} src='http://via.placeholder.com/48x48' circular />
      <Form>
        {/* TODO Make this auto expand as text is typed */}
        <Form.TextArea control={TextArea} rows={1} placeholder='Add a public comment...' />
      </Form>
    </div>
  );
}
