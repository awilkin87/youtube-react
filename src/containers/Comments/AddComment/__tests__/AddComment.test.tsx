import { shallow } from 'enzyme'
import AddComment from '../AddComment'

test('Render AddComment', () => {
  const wrapper = shallow(<AddComment />);
  expect(wrapper).toMatchSnapshot();
});
