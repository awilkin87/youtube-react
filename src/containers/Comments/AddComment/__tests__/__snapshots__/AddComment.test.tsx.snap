// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render AddComment 1`] = `
<div
  className="container"
>
  <Image
    as="img"
    circular={true}
    className="userAvatar"
    src="http://via.placeholder.com/48x48"
    ui={true}
  />
  <Form
    as="form"
  >
    <FormTextArea
      as={[Function]}
      control={[Function]}
      placeholder="Add a public comment..."
      rows={1}
    />
  </Form>
</div>
`;
